docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+{ .annotate }
+
+1. !!! tip "Tip: Windows WSL users"
+
+        If you've already set up the AWS CLI in Windows, WSL will link to the configuration files located in the Windows
+        location. Your credentials will also be linked to the `.aws/` folder that should now be showing in your VSCode
+        Explorer tab. Edit as necessary, but note this CDK expects a profile called `tsdat` to function.
+
+        You can check that the profile was set properly using:
+        
+        `aws configure list`
+
+        If you're using a profile different from `tsdat`, set the new one using
+        
+        `export AWS_PROFILE=<new_profile>`
+
+        The new profile name should now show up in the output of `aws configure list`
+
+```txt title="~/.aws/credentials"
+[tsdat]
+aws_access_key_id=XXXXXXX
+aws_secret_access_key=XXXXXX
+aws_session_token=XXXXXX
+```
+
+### Bootstrap AWS resources
+
+!!! warning
+
+    Check your [CloudFormation stacks](https://us-west-2.console.aws.amazon.com/cloudformation/home?region=us-west-2)
+    first to see if you need to deploy the bootstrap. If you see a stack named `CDKToolkit` then you can SKIP this step.
+
+    This should only be run ONCE for your AWS Account/Region. It won't break anything if you run it more than once, but
+    it's just not recommended.
+
+```shell
+cd aws-template
+./bootstrap_cdk.sh
+```
+
+Bootstrapping is the process of provisioning resources for the AWS CDK before you can deploy AWS CDK apps into an
+AWS environment. An AWS environment is a combination of an AWS account and region.
+
+These resources include an Amazon S3 bucket for storing files and IAM roles that grant permissions needed to perform
+deployments.
+
+The required resources are defined in an AWS CloudFormation stack, called the bootstrap stack, which is usually named
+CDKToolkit. Like any AWS CloudFormation stack, it appears in the AWS CloudFormation console once it has been deployed.
+
+### Deploy the stack
+
+You can re-run this for each branch you want to deploy (e.g., dev, prod, etc.) and any time you make changes to the
+stack (e.g., if you add a new permission to your lambda role).
+
+Most deployments will not need to change anything in the stack, but advanced users are free to customize.
+
+!!! note
+
+    You will need to commit and push all of your changes for this to work correctly.
+
+```shell
+cd aws-template
+./deploy_stack.sh main  # (1)!
+```
+
+1. Here `main` refers to the `main` branch of the `pipeline-template` repo. We recommend deploying the `main` branch
+because it is slightly easier to maintain. You could also create a `release` branch and deploy that instead if you
+prefer to have a separate branch for production releases.
+
+!!! tip
+
+    The very first time you run `./deploy_stack.sh` for a given branch you will need to manually release a CodePipeline
+    change in AWS to get it to build the initial container images and lambda functions.
+
+## Deploying Pipeline Changes
+
```

Comment:
The term 'pipeline' is confusing because we have our Tsdat pipelines and also a CodePipeline from AWS.  Perhaps we should change section headers and text to avoid confusion.  For example:  "Deploying Tsdat Pipeline Changes", "Adding a new Tsdat pipeline", etc.

Reply:
Yeah, that term is definitely overloaded. Imo, "Tsdat Pipeline Changes" is still a bit vague because all the repos are "tsdat" to our users, and they aren't editing the `tsdat/tsdat` repo. I changed it to "Deploying `pipeline-template` Changes" and `Adding a new ingest or VAP pipeline" instead.